README.md

```diff
@@ -15,6 +15,10 @@ You can override cursor highlighting by defining `CursorWord` group and disablin
 
 To lower the time it takes to make the cursorline appear, set `vim.g.cursorline_timeout` (lua) or `g:cursorline_timeout` (vimscript), where `1000` is the default value in milliseconds.
 
+To disable the cursorline set `vim.g.cursorline_highlight` (lua) `g:cursorline_highlight` (vimscript) to `false`.
+
+To disable the cursorline or cursorword on certain filetypes or buffers set `vim.g.cursorline_disabled_filetypes`, `vim.g.cursorline_disabled_buftypes`, `vim.g.cursorword_diesabled_filetypes`, `vim.g.cursorword_diesabled_buftypes` (lua) `g:cursorline_disabled_filetypes`, `g:cursorline_disabled_buftypes`, `g: cursorword_diesabled_filetypes`, `g:cursorword_diesabled_buftypes` (vimscript) as tables (for multiple filetypes/buftypes) or strings (for single filetype/buftype).
```

Comment:
I see a typo `diesabled`.
maybe add some example on how to use those values?

Reply:
Sorry for the late reply.
I fixed the typo in the new commit and added an example of usage with one of the variables.